dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <UploadFormSubcategoryHeader label="Activity Name" description="Name of the activity." />
+  <div class="form-field next-to-each-other">
+    <p>
+      <b> {{ selectedActivities ? selectedActivities.name : "" }}</b>
+    </p>
+
+    <FormKit
+      type="hidden"
+      name="activityName"
+      :modelValue="selectedActivities ? selectedActivities.value : ''"
+      disabled="true"
```

Comment:
Why `disabled`?

Reply:
This is a hidden input and by disabling it we prevent it from being sent with the form. It's value `selectedActivities` is used in other parts of the code.

Reply:
I deleted all of disabled="true" on type="hidden" inputs